jobs/clean_cubes.js

```diff
@@ -21,7 +21,11 @@ const DEFAULT_BASICS = [
 const BATCH_SIZE = 1024;
 
 const needsCleaning = (cube) =>
-  !cube.cards || !Array.isArray(cube.basics) || cardsNeedsCleaning(cube.cards) || cardsNeedsCleaning(cube.maybe);
+  !cube.cards ||
+  !Array.isArray(cube.basics) ||
+  cardsNeedsCleaning(cube.cards) ||
+  cardsNeedsCleaning(cube.maybe) ||
+  cube.tags.some((tag) => !tag || tag.toLowerCase() !== tag);
```

Comment:
Do we really want to disallow tags with uppercase characters?

Reply:
If anything, I'd argue that a better check here would be one that would clean up leading/trailing whitespace.

Reply:
(I just realized this is for cube tags and not card tags, so it isn't that big of a deal. We should still remove the whitespace though.)

Reply:
Yes, because it makes querying by tags either much more expensive, or makes it case sensitive